doc_src/index.hdr.in

```diff
@@ -1050,7 +1050,7 @@ function fish_user_key_bindings
 end
 \endfish
 
-When in vi-mode, the <a href="fish_mode_prompt.html">`fish_mode_prompt`</a> function will display a mode indicator to the left of the prompt. The `fish_vi_cursor` function will be used to change the cursor's shape depending on the mode in supported terminals. To disable this feature, override it with an empty function.
+When in vi-mode, the <a href="fish_mode_prompt.html">`fish_mode_prompt`</a> function will display a mode indicator to the left of the prompt. The `fish_vi_cursor` function will be used to change the cursor's shape depending on the mode in supported terminals. To disable this feature, override it with an empty function. Currently `fish_mode_prompt` calls `fish_default_mode_prompt`, this means you could use `fish_default_mode_prompt` to display the cursor or default cursor somewhere else.
```

Comment:
Drop the "currently". The documentation describes how things are, unless we have explicit plans to change it. In this case we don't.

We want to make a bunch of things clear here:

- fish_mode_prompt can be changed to an empty function to disable the feature

- fish_mode_prompt can be changed to something else to display it differently

- fish_default_mode_prompt can be used elsewhere to display the mode in other places - not at the beginning of the line

This also has nothing to do with a "cursor", that would be `fish_vi_cursor`, which is a different feature.

So how about this:

"To display the mode elsewhere (like in your right prompt), use the output of the `fish_default_mode_prompt` function"?